pkg/auth/defaultmanager.go

```diff
@@ -12,27 +12,23 @@ type DefaultTokenParserConfiguration interface {
 	GetEnvironment() string
 }
 
-var muKM sync.Mutex
-var muTP sync.Mutex
+var (
+	initDefaultTokenParserOnce *sync.Once = &sync.Once{}
 
-var defaultKeyManagerHolder *KeyManager
-var defaultTokenParserHolder *TokenParser
+	defaultKeyManagerHolder  *KeyManager
```

Comment:
why do we have those `Holder` suffixes? What do they mean?

Reply:
I have no idea but I personally don't see any particular reason. That code was not created by me. I'm just slightly re-factoring and re-formatting it so it can be used by the proxy too.